exts/tex.py

```diff
@@ -0,0 +1,65 @@
+"""
+TeX extension for qrm
+---
+Copyright (C) 2021 thxo
+
+This file is part of qrm2 and is released under the terms of
+the GNU General Public License, version 2.
+"""
+
+import aiohttp
+from io import BytesIO
+from urllib.parse import urljoin
+
+import discord
+import discord.ext.commands as commands
+
+import common as cmn
+import data.options as opt
+
+
+class TexCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.session = aiohttp.ClientSession(connector=bot.qrm.connector)
+        self.rtex_instance = getattr(opt, "rtex_instance", "https://rtex.probablyaweb.site/")
```

Comment:
For now all config keys are hard-required
We're going to explore options (#372), but this line will bite us by surprise when moving out of the current system